.github/workflows/test-e2e-gpu.yaml

```diff
@@ -0,0 +1,92 @@
+name: GPU E2E Test
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, labeled]
+
+jobs:
+  gpu-e2e-test:
+    name: GPU E2E Test
+    runs-on: oracle-vm-16cpu-a10gpu-240gb
+
+    env:
+      GOPATH: ${{ github.workspace }}/go
+    defaults:
+      run:
+        working-directory: ${{ env.GOPATH }}/src/github.com/kubeflow/trainer
+
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes-version: ["1.33.1"]
+
+    steps:
+      - name: Check GPU label
```

Comment:
Did you push this commit: https://github.com/kubeflow/trainer/pull/2762#discussion_r2314225326 ?

Reply:
Not exactly that, since still it wasnt exiting, but executing the remaining steps. Added both exit and flag for that.

Reply:
Ok, we can keep it for now.